tools/build_script_generator/scons/site_tools/bitmap.py

```diff
@@ -24,26 +24,19 @@ def bitmap_action(target, source, env):
 	bitmap.convert(image=str(source[0]),
 	               outpath=dirname(str(target[0])))
 
-def bitmap_string(target, source, env):
-    return "{0}Create Bitmap·· {1}{3}{2}" \
-           .format("\033[;0;32m", "\033[;0;33m", "\033[;0;0m", str(target[0]))
-
 def bitmap_emitter(target, source, env):
 	header = splitext(str(target[0]))[0] + ".hpp"
 	target.append(header)
 	return (target, source)
 
 # -----------------------------------------------------------------------------
 def generate(env, **kw):
-	env.Append(
-		BUILDERS = {
-			'Bitmap': env.Builder(
-				action = env.Action(bitmap_action, bitmap_string),
-				suffix = '.cpp',
-				src_suffix = '.pbm',
-				emitter = bitmap_emitter,
-				single_source = True),
-	})
+	env['BUILDERS']['Bitmap'] = env.Builder(
+		action = Action(bitmap_action, cmdstr="$BITMAPCOMSTR"),
+		suffix = '.cpp',
+		src_suffix = '.pbm',
+		emitter = bitmap_emitter,
+		single_source = True)
```

Comment:
The reason for these changes is that `env.Action()` did not resolve the COMSTR properly (just blank `$SOURCE` and `$TARGET`), hence the use of a function before, while using `SCons.Action.Action()` (or just `Action()` via import) did resolve this properly. I couldn't find this as documented behavior, perhaps it's a bug, perhaps not… who knows with SCons.